dashboard/src/components/AppView/AppView.tsx

```diff
@@ -35,7 +35,7 @@ export interface IAppViewProps {
 }
 
 interface IAppViewState {
-  deployments: Array<IKubeItem<IResource>>;
+  deployRefs: ResourceRef[];
```

Comment:
I think in the future we can combine all the refs and then filter by kind, though not sure if that's important.